Cargo.toml

```diff
@@ -11,4 +11,4 @@ documentation = "http://doc.servo.org/fnv/"
 [lib]
 name = "fnv"
 path = "lib.rs"
-doctest = false
```

Comment:
Any reason why doctests were turned off?

FYI, you can add code that doesn't get rendered by putting a `#` in front. See this section for more information:

https://doc.rust-lang.org/book/documentation.html#documentation-as-tests